src/main/resources/templates/userHomepage.html

```diff
@@ -5,8 +5,14 @@
     <title>Amazin Main</title>
 </head>
 <body>
-<p>User Login Successful, Welcome to Book Store</p>
-<fieldset>
```

Comment:
You removed this line "<fieldset>" so it seems there is a missin starting tag for the ending. fix this as well